### ISSUE TITLE
Trusts: Lua runtime error: Unexpected character line 7

### ISSUE THREAD

**Issue by Millercan66** [Windower/Issues#1083]
Downloaded Windower 4.3 stable version. Trust addon is useless. Checked the internet on how to fix problem, no luck. I'm not a coder I'm just a FFXI player. I don't understand the whole Lua thing but this problem has prevented me from saving trust sets, listing trust sets, and recalling them. If you could sent me instructions on how to fix this I would be grateful. gncmill@gmail.com. 

The whole line in the BG console reads:

Trusts: Lua runtime error: libs/config.lua:99: XML error, line 7: unexpected character '#'.

**Comment by Nifim** [comment]
What is the name you tried to save the set under? it looks like you probably saved a set with a `#` in it and that is not a legal character for a set name. you can delete your settings file located in the `addon/trust` folder and then try saving the settings again using a simple name like `ddparty` or `supportparty`

**Comment by RubenatorX** [comment]
yeah, you corrupted your settings file by using special characters in your set name. keep it simple and it won't break itself

**Comment by Millercan66** [comment]
Thanks, how do I get to said folder?

**Comment by RubenatorX** [comment]
`/path/to/windower/addons/trusts/data/settings.xml`

**Comment by z16** [comment]
This is really a config issue. It's not serializing special characters correctly, but it should. The XML library handles that case correctly, but the config library only uses that library to *read* settings, not to *write* them, because it wouldn't support keeping certain formatting structures (like comments).

To fix this, we would need to change a few lines in the `nest_xml` functions to escape the values they're formatting. It's not super trivial but can be done in a short ish time if someone wants to take a stab at it. Poke me if you want to and need some pointers.

**Comment by ghost** [comment]
im having this issue how do i fix this??

**Comment by RubenatorX** [comment]
> im having this issue how do i fix this??

delete the addon's settings file and reload the addon, and don't use special characters such as `<` or  `>` in your set names

**Comment by Nifim** [comment]
We commonly issue this advice to avoid special characters in the set names for the trust addon in our support discord, a more generic "settings library does not save special characters properly" should be created.
